### DISCUSSION TITLE
Is there a way to get NIC mac address information by V?

### DISCUSSION THREAD

**Discussion by guizhenwei** [vlang/v#8349] [General]
Hi guys, is there a way to get mac address info by using V's standard library module?

I see it's easy to get that information in Golang, by using `net` lib of Go.

But i didn't find a way to get that information in V.

Thanks for help:)

**Comment by spytheman** [comment]
There is not for now.

**Reply by guizhenwei** [reply]
Understood. Still very early stage.

Thanks for your reply.
